styles.css

```diff
@@ -656,10 +768,19 @@ ul:nth-child(4) {
     font-weight: 500;
     font-size: 15px;
     line-height: 20px;
-    color: #344563;
     padding: 10px 12px;
   }
 
+  .bdl-el a {
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .bdl-el a:hover {
+    color: #5462e0;
+    text-decoration: underline;
+  }
+
```

Comment:
- [ ] The two buttons in the popup window (see the project and live version) should be styled according to the guidelines in the Figma design. Please add the proper styles to them. Remember to add the respective icons 💡 😉 

**Your design**
![image](https://user-images.githubusercontent.com/90222110/188994335-6a0e2d2a-7796-4612-a798-f6e5f377bdbb.png)

**Figma design**
![image](https://user-images.githubusercontent.com/90222110/188994380-ff60d254-841e-4e67-9b5d-b2c0d0ff45b8.png)